nix/ssh-keys.nix

```diff
@@ -1,4 +1,10 @@
 {
   tmplt = "ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBBrsSG9or4lj7j/aBwzIYX98m8kHD78ulKxhNnThWkInoK8iDGdrBzBkLdXRDneTxDWpqZEQBUyIAX1ZTkvluNU= tmplt@perscitia-2019-11-13";
   simpen = "ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBNmYY8kvkqscucREcHBnUZr+3k3icyO8SVjL4Div2WZ2XnvxHoJgj4DT06qTgSEzed8l4iewr7xAg9gk4BFqd9w= ecdsa-key-20201008";
+  Alex=---- BEGIN SSH2 PUBLIC KEY ----
+Comment: "ecdsa-key-20201009"
+AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBI+56U0NQXwd
+7WKUUrc3rFn8DouhhD9SgcNyeBd8CkPWlNdgrTikIcd53IaK3a6z6pm1gsdDcSZR
+ofUqe/EDT/U=
+---- END SSH2 PUBLIC KEY ----
```

Comment:
This is a Putty-specific key; the SSH server does not understand it. The required format is `{key-type} {key} {key-comment}`. There should be an option to copy the key on that format in Putty.